src/runtime-rs/crates/persist/src/network.rs

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2019-2022 Alibaba Cloud
+// Copyright (c) 2019-2022 Ant Group
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Clone, Default)]
+pub struct PhysicalEndpointState {
```

Comment:
Recommended to move the persist state to the corresponding implementation directory, such as `PhysicalEndpointState` to the `PhysicalEndpoint` implementation directory.

Reply:
fixed